compiler/src/main/kotlin/edu/cornell/cs/apl/viaduct/codegeneration/BackendCodeGenerator.kt

```diff
@@ -69,19 +69,14 @@ private class BackendCodeGenerator(
         codeGeneratorMap = initGeneratorMap
     }
 
-    fun generateHostFunction(
+    // generate code for [this.host]'s role in the function named [funName]
+    fun generateFunction(
         host: Host,
```

Comment:
This method doesn't need `host` because the class has it. It also doesn't need `funName` because `functionDeclaration` has it.